Source/RapyutaSimulationPlugins/Public/Core/RRAssetUtils.h

```diff
@@ -343,20 +387,18 @@ class RAPYUTASIMULATIONPLUGINS_API URRAssetUtils : public UBlueprintFunctionLibr
         UClass* blueprintGeneratedClass = nullptr;
         kismetCompilerModule.GetBlueprintTypesForClass(InParentClass, blueprintClass, blueprintGeneratedClass);
 
-        // 2- Create package for the class asset
-        FString packageName = FString::Printf(TEXT("/Game/Blueprints/%s"), *InClassName.ToString());
-        FString assetName;
-        assetToolsModule.Get().CreateUniqueAssetName(packageName, TEXT(""), packageName, assetName);
-        UPackage* package = CreatePackage(*packageName);
-        check(package);
+        // 2- Create blueprint package for the class asset
+        const FString bpClassName = FString::Printf(TEXT("BP_%s"), *InClassName.ToString());
+        FString bpPackageName = FString::Printf(TEXT("/Game/Blueprints/%s"), *bpClassName);

```

Comment:
It might be better to pass the package path as a parameter, and then allow it to be configured elsewhere by the user.